src/helpers/useFetchAdvices.ts

```diff
@@ -0,0 +1,19 @@
+import { useEffect, useState } from "react";
+import { getAdvices } from "./getAdvices";
+import { Advice } from "../types/AdviceTypes";
+
+export const useFetchAdvices = () => {
+
+  const [adviceState, setAdviceState] = useState<Advice>();
+
+  const getTips = async () => {
+    const newAdvice = await getAdvices();
```

Comment:
Es bueno siempre cachar las exepciones (errores) cuando se hacen llamados asíncronos.

```suggestion
    try {
       const newAdvice = await getAdvices();
       setAdvice(newAdvice)
    } catch (e) {
    // Tal vez no cuentes con el schema del objecto error, inclusive no se use el mismo, 
    // pero puedes basarte en esta propiedad. Si el valor es truthy, entonces puedes pintar 
    // en pantalla de que hubo un error
       setError(e) 
    }
```